scripts/getnomad.sh

```diff
@@ -5,7 +5,7 @@
 
 set -e
 
-NOMAD_VERSION=1.5.6
+NOMAD_VERSION=1.5.0
```

Comment:
Bumping past `1.5.1` causes `TestResourceJob_serviceDeploymentInfo` to fail. I'm still investigating why.